result/isisiihs.yaml

```diff
@@ -0,0 +1,378 @@
+            - restarting: Is in Restarting state when Restarting Tlv has been received with RR bits set.
+          type: string
+          x-field-uid: 1
+          x-enum:
+            running:
+              x-field-uid: 1
+            starting:
+              x-field-uid: 2
+            restarting:
+              x-field-uid: 3
+        last_attempt_status:
+          description: >-
+            This container holds the information of the last Graceful Restart initiated 
+            by the neighbor since the adjacency was established.
+          $ref: '#/components/schemas/IsisIIH.NeighborGRLastAttemptStatus'
+          x-field-uid: 2
+
+    IsisIIH.LocalGRLastAttemptStatus:
+      description: |-
+        This object contains the status of the last attempted Graceful Restart status of this router.
+        - succeeded: Choice is set if the last Graceful Status is successful.
+        - failed: The last Graceful Status is unsuccessful.
+        - inprogress: The last Graceful Restart status is in progress.
+        - unavailable: The last Graceful Restart status is not initiated.
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          x-enum:
+            succeeded:
+              x-field-uid: 1
+            failed:
+              x-field-uid: 2
+            inprogress:
+              x-field-uid: 3
+            unavailable:
+              x-field-uid: 4
+        succeeded:
+          $ref: '#/components/schemas/IsisIIH.LocalGRLastAttemptSucceeded'
+          x-field-uid: 2
+        failed:
+          $ref: '#/components/schemas/IsisIIH.LocalGRLastAttemptFailed'
+          x-field-uid: 3
+        inprogress:
+          type: boolean
+          x-field-uid: 4
+        unavailable:
+          type: boolean
+          x-field-uid: 5
+
+    IsisIIH.LocalGRLastAttemptFailed:
+      description: >-
+        This container contains the failure status of the last Graceful Restart initiated by this router.
+      type: object
+      properties:
+        reason:
+          description: |-
+            Failure reason of last Graceful Restart.
+          type: string
+          x-field-uid: 1
+
+    IsisIIH.LocalGRLastAttemptSucceeded:
+      description: >-
+        This container contains details about the successful status of the last Graceful Restart initiated by this router.
+      type: object
+      properties:
+        lsdb_syncup_time:
+          description: |-
+            The time is taken to synchronize the L1 and L2 LSDB by this Restarting router.
+            By this time, the CSNP list is cleared and all LSPs are collected by the neighbor(s).
+          type: integer
+          format: uint32
+          x-field-uid: 1
+        adjacency_bring_up_time:
+          description: |-
+            The time (in second) is taken to bring up the adjacency by this Restarting router.
+            The time is counted when the Restart TLV with RR bit set in a IIH PDU is sent 
```

Comment:
is measured from when the Restart TLV with RR bit set is sent in an IIH PDU upto the time when Restart TLV is sent with RR bit unset.